gradle.properties

```diff
@@ -1,4 +1,4 @@
-release_version = 2.16.1
+release_version = 2.17.0-TH2-2194.1
```

Comment:
Why does the version have the suffix? The workflow for building a dev release will do it for you

Reply:
Oh, sorry for confusing, my bad.